conf/buildout.cfg

```diff
@@ -111,17 +112,18 @@ gpxpy = 1.1.2
 BeautifulSoup4 = 4.6.0
 requests = 2.18.4
 django-shapes = 0.2.0
-django-crispy-forms = 1.6.1
+django-floppyforms = 1.7.0
+django-crispy-forms = 1.7.2
 django-compressor = 2.2
 django-filter = 1.1.0
 easy-thumbnails = 2.5.0
-django-tinymce = 2.4.0
-psycopg2 = 2.6.2
+django-tinymce = 2.5.0
+psycopg2 = 2.7.4
```

Comment:
Il faudrait laisser 2.6.2 uniquement pour supprimer le warning `UserWarning: The psycopg2 wheel package will be renamed from release 2.8`